src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -126,13 +126,9 @@ def get_nwb_imaging_metadata(imgextractor: ImagingExtractor):
                     description="An optical channel of the microscope.",
                 )
             )
-    # set imaging plane rate:
-    rate = np.nan if imgextractor.get_sampling_frequency() is None else float(imgextractor.get_sampling_frequency())
 
-    # adding imaging_rate:
-    metadata["Ophys"]["ImagingPlane"][0].update(imaging_rate=rate)
```

Comment:
Should this be also done in `get_nwb_segmentation_metadata`?

Reply:
Yep, thanks for catching that

Reply:
Indeed. Thanks.